plugins/modules/ec2_eni.py

```diff
@@ -668,26 +666,26 @@ def delete_eni(connection, module):
     try:
         if force_detach is True:
             if "Attachment" in eni:
-                connection.detach_network_interface(
-                    aws_retry=True,
+                detach_network_interface(
+                    connection,
```

Comment:
The params here and on line 701 need to be updated to snake case to match the module_util function, that should fix the failing integration test.